ktor-server/ktor-server-core/api/ktor-server-core.api

```diff
@@ -16,12 +16,11 @@ public abstract interface class io/ktor/application/ApplicationCall {
 public class io/ktor/application/ApplicationCallPipeline : io/ktor/util/pipeline/Pipeline {
 	public static final field ApplicationPhase Lio/ktor/application/ApplicationCallPipeline$ApplicationPhase;
 	public fun <init> ()V
-	public fun <init> (ZLio/ktor/server/application/ApplicationEnvironment;)V
-	public synthetic fun <init> (ZLio/ktor/server/application/ApplicationEnvironment;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (Z)V
```

Comment:
is it intended?

Reply:
Will check, thank you

Reply:
That's strange, cause there were no changes in this classes